docs/devguide/annex.rst

```diff
@@ -0,0 +1,37 @@
+.. _annex:
+
+#####
+Annex
+#####
+
+Documentation that doesn't fit with other sections, but isn't big enough to justify having a separate section.
+
+.. _fnmatch-exprs:
+
+*******************
+Fnmatch Expressions
+*******************
+
+A wildcard-capable string used to match one or more names from a set.
+This is used in :ref:`qos-partition` and in :ref:`sec` documents.
+Recognized values will conform to POSIX ``fnmatch()`` function as specified in POSIX 1003.2-1992, Section B.6.
```

Comment:
This could link to POSIX spec from The Open Group

Reply:
I think was going to when I originally wrote this for security, but I couldn't figure out what to link to. https://pubs.opengroup.org/onlinepubs/9695969399/toc.pdf doesn't have a section B. It mentions `fnmatch` and talks about what is basically the same pattern matching in section 2.13, but I didn't know if that was enough. I ended up just documenting the ACE implementation of `fnmatch` since that's what we are using anyway.